services/libs/tracing/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@crowd/tracing",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "build": "rm -rf dist && ./node_modules/.bin/tsc",
```

Comment:
Please merge `main` into this branch - we stopped with the building of libraries - also check other libraries tsconfig.json after merging - there is a difference now - two properties were removed and `@/` path mapping has also been removed as it doesn't really work without building the libraries when referencing a library that has such path mapping.

Reply:
Also build_libs.sh and watch_libs.sh scripts were removed on main.